src/app/user/painting/component/painting-details/painting-details.component.ts

```diff
@@ -0,0 +1,77 @@
+import {Component, OnInit} from '@angular/core';
+import {PaintingService} from '../../service/painting.service';
+import {ActivatedRoute} from '@angular/router';
+import {ArtistService} from '../../../artist/service/artist.service';
+import {ArtistDetails} from '../../../artist/entity/artist-details';
+import {PaintingDetails} from '../../entity/painting-details';
+
+@Component({
+  selector: 'app-painting-details',
+  templateUrl: './painting-details.component.html',
+  styleUrls: ['./painting-details.component.scss']
+})
+export class PaintingDetailsComponent implements OnInit {
+  private painting: PaintingDetails;
+  private artist: ArtistDetails;
+
+  constructor(private paintingService: PaintingService,
+              private artistService: ArtistService,
```

Comment:
I use ArtistService with PaintingService because we're depending on [artistId] that fetching from [getPainting] method to get the artist details
anyway, I'll try it if we still have time.